core/profiling/profiling.h

```diff
@@ -104,17 +136,26 @@ struct PerfettoGroupedEventEnder {
 
 #define GodotProfileAlloc(m_ptr, m_size)
 #define GodotProfileFree(m_ptr)
+#define GodotProfileZoneScript(m_varname, m_zone_name, m_function, m_file, m_line, m_color) \\ TODO
+
 void godot_init_profiler();
+void godot_cleanup_profiler();
 
 #else
 // No profiling; all macros are stubs.
 
 void godot_init_profiler();
+void godot_cleanup_profiler();
 
 // Tell the profiling backend that a new frame has started.
 #define GodotProfileFrameMark
 // Defines a profile zone from here to the end of the scope.
 #define GodotProfileZone(m_zone_name)
+// Defines a profile zone from here to the end of the scope, with a variable
+// name specified.
+#define GodotProfileZoneV(m_varname, m_zone_name)
```

Comment:
Would it be possible to use the existing group macros instead? If possible, I'd like to avoid adding too many similar macros.